.ado/pipelines/config/workloads.json

```diff
@@ -0,0 +1,35 @@
+{
+    "catalogservice": {
+        "chart": {
+            "name": "afintglobalcr.azurecr.io/helm/catalogservice",
```

Comment:
Any specific reason to use a different schema for charts and images?

Reply:
not really. I was just not sure if the name for helm charts will always be in the same format as for images? (registry/name)